### PR TITLE
Qing cloud

### PR DESCRIPTION
**Pull request by FURIOUS-7**
Hello,

Since ClearLinux image has been officially uploaded and published on QingCloud.com(CSP in China), we decided to make an instruction on how to create ClearLinux VM on QingCloud.com. Since QingCloud provides public cloud services mainly in China, so the instruction is written in Chinese.

Changes:
1. Added pngs
2. Added qingcloud.rst

Thanks!

Best Regards,
Xiaowei Wu

### CODE REVIEW

**Comment by mvincerx** [comment]
@ttzeng Can you please validate this guide and offer any feedback?

**Comment by mvincerx** [comment]
@ttzeng. Do you have time to validate this?

**Comment by mvincerx** [comment]
FYI: Please disregard the failure of Greetings above. It's unrelated to this PR. It's on my list.

**Comment by mvincerx** [comment]
@linjiach, do you have bandwidth to review the mandarin content in this PR?  It's only about 2 pages.

**Comment by linjiach** [comment]
Yes, I will go over it.

**Comment by mvincerx** [comment]
Thanks!, @linjiach

File: source/get-started/cloud-install/qingcloud.rst [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+.. contents::
+   :local:
+   :depth: 1
+
+必备条件
+#############
+
+本教程假定您已经完成了如下默认配置：
+
+* 您的环境可以运行 SSH 以访问远程 Clear Linux OS 虚拟镜像主机。
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

File: source/get-started/cloud-install/qingcloud.rst [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+* 您知悉浏览器下载文件的绝对路径。
+* 您已设置了 QingCloud 的用户帐户，并确保该账户为启用状态，并且已登录到 QingCloud 控制台。 要了解有关青云和设置账户的更多信息，请访问青云官网，网址为 https://www.qingcloud.com/。
+
+在 QingCloud 控制台中选择并启动 Clear Linux OS 主机
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

File: source/get-started/cloud-install/qingcloud.rst [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+      :scale: 50 %
+      :alt: QingCloud 控制台
+
+      图1: QingCloud 控制台
+    
+   选择此选项后，页面将跳转到“创建主机”页面。
+
+#. 在创建主机页面，点击图2中所示的 **“系统”** 选项
+
+   .. figure:: /_figures/qingcloud/QingCloud-2.png
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: The image is a bit confusing. I think it's ok to NOT show this at all, and add the highlight of "系统" in the next one. This image is actually showing what is in "常用" and doesn't have Clear Linux in there. Readers might panic a little.

File: source/get-started/cloud-install/qingcloud.rst [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+      :scale: 50 %
+      :alt: QingCloud 创建主机
+
+      图2: QingCloud 创建主机
+
+   选择此选项后，页面将显示出不同类型的操作系统。
+
+#. 在系统选择界面，点击图3中所示的最右侧 **“Clear Linux”** 图标，选择 **Clear Linux OS** ，并点击 **“下一步”** 按钮
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: If remove 图2 this sentence needs to be changes too.

File: source/get-started/cloud-install/qingcloud.rst [hunk 5]
```diff
@@ -0,0 +1,245 @@
+.. _qingcloud:
+
+如何在青云 QingCloud\* 上创建 Clear Linux\* OS 主机
```
[the remark this answers is not in the document]
Comment (linjiach) [reply source/get-started/cloud-install/qingcloud.rst]: 主机 -> 虚拟机？

File: source/get-started/cloud-install/qingcloud.rst [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+   图18: 确认主机当前处于正常状态
+
+#. 复制当前 Clear Linux OS 主机的公网IP地址，并使用 SSH 客户端进行连接。 这里我们需要用到之前保存的 SSH 密钥。
+#. 在此教程中，以 MobaXterm 客户端为例演示登录过程。请如图19检查各项。用户名我们选择 **root**，密钥请选择之前下载并保存到本地的 SSH 密钥。
+
+   .. figure:: /_figures/qingcloud/QingCloud-19.png
+      :scale: 50 %
+      :alt: SSH 登录主机设置
+
+      图19: SSH 登录主机设置
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: the IP address in this screenshot is censored, but not everywhere else. Either display it or censor the IP address in other screenshots too.

File: source/get-started/cloud-install/qingcloud.rst [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+      :scale: 50 %
+      :alt: SSH 登录成功
+
+      图20: SSH 登录成功
+
+
+删除 Clear Linux 主机
+##################################
+
+本章节介绍如何在 QingCloud 上删除所创建的 Clear Linux OS 主机。
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

File: source/get-started/cloud-install/qingcloud.rst [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+
+#. 通过左侧导航栏依次选择 **“计算”**、**“主机”** 后，找到刚刚创建的 Clear Linux OS 主机，如图21所示选中此主机，再点击上方 **“更多操作”** 按钮选择 **“删除”**，即可删除主机。
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

File: source/get-started/cloud-install/qingcloud.rst [hunk 9]
```diff
@@ -0,0 +1,245 @@
...
+#. 复制当前 Clear Linux OS 主机的公网IP地址，并使用 SSH 客户端进行连接。 这里我们需要用到之前保存的 SSH 密钥。
+#. 在此教程中，以 MobaXterm 客户端为例演示登录过程。请如图19检查各项。用户名我们选择 **root**，密钥请选择之前下载并保存到本地的 SSH 密钥。
+
+   .. figure:: /_figures/qingcloud/QingCloud-19.png
+      :scale: 50 %
+      :alt: SSH 登录主机设置
+
+      图19: SSH 登录主机设置
+
+#. 设置成功后，点击登录即可登录到 Clear Linux OS 主机。
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

File: source/get-started/cloud-install/qingcloud.rst [hunk 10]
```diff
@@ -0,0 +1,245 @@
...
+
+请您点击导航栏左侧 **“计算”**、**“主机”** 按钮，确认当前主机处于正在运行状态，且已绑定了公网IP地址。如图18所示。
+
+.. figure:: /_figures/qingcloud/QingCloud-18.png
+   :scale: 50 %
+   :alt: 确认主机当前处于正常状态
+
+   图18: 确认主机当前处于正常状态
+
+#. 复制当前 Clear Linux OS 主机的公网IP地址，并使用 SSH 客户端进行连接。 这里我们需要用到之前保存的 SSH 密钥。
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

File: source/get-started/cloud-install/qingcloud.rst [hunk 11]
```diff
@@ -0,0 +1,245 @@
...
+      图16: 提交绑定
+
+   .. figure:: /_figures/qingcloud/QingCloud-17.png
+      :scale: 50 %
+      :alt: 公网IP绑定成功
+
+      图17: 公网IP绑定成功
+
+
+连接到 Clear Linux OS 主机
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

File: source/get-started/cloud-install/qingcloud.rst [hunk 12]
```diff
@@ -0,0 +1,245 @@
...
+
+#. 之后如图14通过导航栏点击 **“计算”**、**“网卡”** 按钮来到网卡界面。
+
+   .. figure:: /_figures/qingcloud/QingCloud-14.png
+      :scale: 50 %
+      :alt: 网卡界面
+
+      图14: 网卡界面
+
+#. 在网卡界面，按照图15选中刚刚创建的 Clear Linux OS 主机的网卡，并点击上方 **“更多操作”** 按钮，再点击 **“绑定公网IPv4”** 按钮。
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

File: source/get-started/cloud-install/qingcloud.rst [hunk 13]
```diff
@@ -0,0 +1,245 @@
...
+
+#. 在确保 SSH 密钥已妥善下载保存的情况下，如图9检查主机的基本信息，确认无误后请点击 **“创建”** 按钮。
+
+   .. figure:: /_figures/qingcloud/QingCloud-9.png
+      :scale: 50 %
+      :alt: 确认信息并创建主机
+
+      图9: 确认信息并创建主机
+
+   确认后，QingCloud将会创建 Clear Linux OS 主机，您可以在新的界面中查看当前主机的状态。
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

File: source/get-started/cloud-install/qingcloud.rst [hunk 14]
```diff
@@ -0,0 +1,245 @@
...
+
+   .. figure:: /_figures/qingcloud/QingCloud-4.png
+      :scale: 50 %
+      :alt: 配置选择
+
+      图4: 配置选择
+
+   在点击 “下一步” 按钮之后，您将来到网络设置界面。
+
+#. 在网络设置界面，您可以创建私有的 VPC 网络，也可以快速测试 Clear Linux OS 选择基础网络。 这里我们选择 **“基础网络”**。
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

File: source/get-started/cloud-install/qingcloud.rst [hunk 15]
```diff
@@ -0,0 +1,245 @@
...
+
+   选择此选项后，页面将显示出不同类型的操作系统。
+
+#. 在系统选择界面，点击图3中所示的最右侧 **“Clear Linux”** 图标，选择 **Clear Linux OS** ，并点击 **“下一步”** 按钮
+
+   .. figure:: /_figures/qingcloud/QingCloud-3.png
+      :scale: 50 %
+      :alt: 选择 Clear Linux OS
+
+      图3: 选择 Clear Linux OS
```
Comment (linjiach) [inline source/get-started/cloud-install/qingcloud.rst]: Change "Clear Linux OS"to "|CL|"

**Comment by mvincerx** [comment]
@linjiach, Thank you for reviewing this. 
@FURIOUS-7, will you please apply the requested changes?

File: source/get-started/cloud-install/qingcloud.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (FURIOUS-7) [reply source/get-started/cloud-install/qingcloud.rst]: 1. "Clear Linux OS" changed to "|CL|", done.
2. 主机 changed to 虚拟主机

File: source/get-started/cloud-install/qingcloud.rst (hunk 3 above)
Comment (FURIOUS-7) [reply source/get-started/cloud-install/qingcloud.rst]: Thanks for pointing this out. Already done.

File: source/get-started/cloud-install/qingcloud.rst (hunk 6 above)
Comment (FURIOUS-7) [reply source/get-started/cloud-install/qingcloud.rst]: Thanks for pointing this out. Done!

**Comment by FURIOUS-7** [comment]
Thanks @linjiach ! I've applied the changes according to Linjia's comments. Please take a look when you have time. Thanks again!

File: source/get-started/cloud-install/qingcloud.rst (hunk 4 above)
Comment (linjiach) [reply source/get-started/cloud-install/qingcloud.rst]: "图3" should be "图2" now
the figure numbers following all need to be updated too. Sorry it's a bit tedious but it'd be good to not have discontinued numbering!

**Comment by linjiach** [comment]
Looking good, just a few more things, See comments above.
Let's remove figure 2 file too since it's not needed.

File: source/get-started/cloud-install/qingcloud.rst (hunk 4 above)
Comment (FURIOUS-7) [reply source/get-started/cloud-install/qingcloud.rst]: Thanks for the reminding, it's already done. Changed all the figures' number in order to make them consecutively.

**Comment by linjiach** [comment]
Looking good! Thanks for making the changes!

**Comment by mvincerx** [comment]
@linjiach, Yes please approve it.  Thanks for your willingness to help with this content.

**Comment by mvincerx** [comment]
@intelkevinputnam. Will you please assure that this is merged and published here?: 
https://github.com/clearlinux/clear-linux-documentation/tree/master/locale/zh_CN/LC_MESSAGES/get-started/cloud-install

This is a one-time request, out-of-cycle build. CL Docs decided not to create an English version of this.  Let me know if you think it's necessary to have a `.po` file for this too.  I'd recommend it for completeness.

**Comment by mvincerx** [comment]
LGTM.

**Comment by intelkevinputnam** [comment]
@mvincerx, @linjiach, @FURIOUS-7 

1. All of the section headers use the top-level header "#" (I can fix this).
2. Can you please provide an outline for the docs in English (just translate section headers)? This would help us when soliciting help to have the entire doc translated when the time comes. 
3. I'm going to change the title of the doc to: |CL-ATTR| on QingCloud\* (如何在青云 QingCloud 上创建 |CL| 虚拟主机) as I intend to publish on the main site before patching it into the Chinese language version.

**Comment by linjiach** [comment]
**Title:**
How to Create Clear Linux OS Virtual Machine on QingCloud
**Section headers:**
Prerequisites
Select and Create Clear Linux OS VM in QingCloud Dashboard
Apply for Public IP and Add to the VM
Connect to Clear Linux OS VM
Delete Clear Linux OS VM
Delete Public IP Applied

**Comment by intelkevinputnam** [comment]
@linjiach Thanks! :-)

**Comment by mvincerx** [comment]
Yes, thanks, @linjiach.

**Comment by linjiach** [comment]
Sorry, I just noticed, but we need to add an entry in this page too:
https://docs.01.org/clearlinux/latest/get-started/index.html#deploy-to-the-cloud

**Pull request merged**